champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/sword/Bullseye.java

```diff
@@ -138,9 +145,14 @@ public void updateCharge() {
                 continue;
             }
             // Spawn particles
+            if (player == null) return;
             if (playerBullsEyeData.getTargetFocused() != null && playerBullsEyeData.getTarget().isValid()) {
-                playerBullsEyeData.spawnFocusingParticles();
+                Gamer gamer = championsManager.getClientManager().search().online(player).getGamer();
```

Comment:
You are already getting the gamer below this, no need to do it twice in the same function. Do it once just after the level check